### PR TITLE
Module 2 day 1 saddam dev

### PR DESCRIPTION
**Pull request by Saddamkhanlavani**
Added html task file and removed saddam_task.txt file under day1 module-2.

### CODE REVIEW

File: Module-2/Day-1/Task/Saddam/saddam_task.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+    
+    var   admin,name; 
+        name="john";
+        admin=name;
+        admin,name;
```
Comment (IshaaqhShaik) [inline Module-2/Day-1/Task/Saddam/saddam_task.html:14]: what does this line do?

File: Module-2/Day-1/Task/Saddam/saddam_task.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        alert(admin); 
+    
+
+    let saddam,khan;
+        saddam=24;
+        khan="lavani";
+        alert(saddam+khan)
+
+    let saddu=[1,2,3];
```
Comment (IshaaqhShaik) [inline Module-2/Day-1/Task/Saddam/saddam_task.html:23]: nothing is being done with this array ?

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]

File: Module-2/Day-1/Task/Saddam/saddam_task.html (hunk 1 above)
Comment (Saddamkhanlavani) [reply Module-2/Day-1/Task/Saddam/saddam_task.html]: I had tried to give declare the variables without giving anything before it and it didn't showed anything in the output as previously i had declared the same variables so. and I forgot to remove that after experimenting Now, I had removed it.

File: Module-2/Day-1/Task/Saddam/saddam_task.html (hunk 2 above)
Comment (Saddamkhanlavani) [reply Module-2/Day-1/Task/Saddam/saddam_task.html:22]: I had made changes to this and added console.log(saddu) as previously after opening live server console, there I had given console.log(saddu) and checked that Iam getting the output as (3) [1, 2, 3]. Now in the code I did the changes.

**Review by IshaaqhShaik — APPROVED** [review]

**Pull request merged**